plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/embeddings/SourceRole.kt

```diff
@@ -13,6 +14,7 @@ sealed interface SourceRole {
     data object ReturnsFalseEffect : SourceRole
     data object ReturnsNullEffect : SourceRole
     data object ReturnsNotNullEffect : SourceRole
+    data class CallsInPlaceEffect(val kind: EventOccurrencesRange = EventOccurrencesRange.UNKNOWN) : SourceRole
```

Comment:
A more specific name would be nice since there are two uses of `CallsInPlace`

Reply:
Also, is the default value needed?  It seems to me like having `UNKNOWN` here would actually not make sense in the error.

Reply:
`CallsInPlaceEffectCallCount`?